kalkulator/src/main/java/no/nav/folketrygdloven/kalkulator/steg/fastsettskjæringstidspunkt/AvklaringsbehovUtlederFastsettBeregningsaktiviteterFP.java

```diff
@@ -84,20 +84,20 @@ public static boolean harFullAAPITilleggTilAnnenAktivitet(BeregningAktivitetAggr
         if (beregningAktivitetAggregat.getAktiviteterPåDato(skjæringstidspunkt).size() <= 1) {
             return false;
         }
-        return hentUtbetalingsprosent(aktørYtelse, skjæringstidspunkt, YtelseType.ARBEIDSAVKLARINGSPENGER)
-                .filter(verdi -> verdi.compareTo(MeldekortUtils.MAX_UTBETALING_PROSENT_AAP_DAG) == 0)
-                .isPresent();
+	    var ytelseFilter = new YtelseFilterDto(aktørYtelse).før(skjæringstidspunkt);
```

Comment:
Her er det potensielt en gremlin - mest synlig for de som begynner i mai pga justering
Siste meldekort før STP henger ikke nødvendigvis under siste vedtak på STP - det siste vedtaket kan være tomt.
MKUtil sisteHeleMK grei - selv om KonfigTjeneste.getMeldekortPeriode() er litt spes med 30 dager ....
Men siden nyligsteMeldekort ikke nødvendigvis henger under nyligsteVedtak så må du enten lete opp vedtaket som matcher MK for å finne forventet MAX - eller propagere ned MAX fra Ytelse til YtelseAnvist

Mulig dette ikke er et problem dersom alle vedtak kommer enten fra Arena eller Kelvin - men er det er en greie ved mixed bestand

Reply:
Situasjonen for de som begynner AAP tidlig mai (eller får endret vedtaket tilbake i tid ellers i året):
* Vedtak v1 med fom=tom som har MK fra ved.fom og med mk.tom > v1.tom. Har 1-2 MK 
* Vedtak v2 med v2.fom=v1.fom sog tom åpen. Har 0..N MK fom vedtaksdato for V2.

Dagens  hentUtbetalingsprosent ser ut til å funke ref MkUtil, selv om finnMeldekortSomGjelderForVedtak (dersom flere MK matcher siste periode) er litt flaky